jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CfnModel.kt

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we get any kind of reference for this file?

Reply:
Added